music/demos/piano_genie.html

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright 2018 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================-->
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <link rel="stylesheet" href="./style.css">
+  <title>Piano Genie</title>
+</head>
+<body>
+  <h1>Piano Genie</h1>
+  <h2>Chris Donahue, Ian Simon, Sander Dieleman</h2>
+
+  <p>Implements our <a href="https://magenta.tensorflow.org/pianogenie">Piano Genie</a> model.</p>
```

Comment:
Looking at the other demos, we tend not to write any authors there, and link to the blog post for more details. I would
- say "implements Magenta's ..." instead of "our"
- add a v short paragraph about what this model does

(you can look at `music_vae.html` or `transcription.html` for an example)